contracts/main/price-feeders/DelayFathomOraclePriceFeed.sol

```diff
@@ -52,48 +50,88 @@ contract DelayFathomOraclePriceFeed is PausableUpgradeable, IFathomOraclePriceFe
     event LogSetPriceLife(address indexed _caller, uint256 _second);
     event LogSetTimeDelay(address indexed _caller, uint256 _second);
 
+    function readPrice() external view override returns (bytes32) {
```

Comment:
2.1.1 Current price is returned in the feed in DelayFathomOraclePriceFeed - fix, now readPrice function returns delayed price instead of current price